lucille-examples/lucille-distributed-example/pom.xml

```diff
@@ -77,7 +77,7 @@
         <executions>
           <execution>
             <id>copy-dependencies</id>
-            <phase>package</phase>
```

Comment:
try moving this to prepare-package instead.  this issue seems only to be triggered inside eclipse , potentially with the newer versions of maven...